cryptography

```diff
@@ -0,0 +1,15 @@
+---
```

Comment:
A few points about the file/folder setup for this PR:

- This file needs to be a Markdown file ending with a `.md` extension. (i.e., `cryptography.md`)
- The `cryptography.md` file needs to be nested in a similarly-named `cryptography` folder.
- The `cryptography` folder (with the nested file) needs to be placed inside the following directory: `content/cybersecurity/concepts`